components/isceobj/Util/ImageUtil/ImageLib.py

```diff
@@ -67,8 +67,8 @@
 
 #######Current list of constants
 constDict = { "PI"  : np.pi,
-              "J"   : np.complex(0.0, 1.0),
-              "I"   : np.complex(0.0, 1.0),
+              "J"   : np.complex64(0.0, 1.0),
+              "I"   : np.complex64(0.0, 1.0),
```

Comment:
Having trouble finding a migration guide for this change, but based on this line in the cpython PR, should this be complex128? https://github.com/numpy/numpy/pull/22607/files#diff-09a2c2e86cca91dac0c336e4dad47197ba3cd837ff6c85abf8525cc3b879d3e6L181

Reply:
@rtburns-jpl I committed your `1j` suggestion, but that'll (I think) be interpreted by `numpy` as `complex128` instead of `complex64`. I think that's likely fine, but if we want it to stay `complex64,` we'll need to explicitly do that.

Reply:
```
              "J"   : np.complex64(1j),
              "I"   : np.complex64(1j),
```
would do it

Reply:
So, since originally this would have been `complex128` via the old numpy alias, I'm content to leave it with the python native type